src/components/pageWrap/PageWrap.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+
+import { polyfillServiceUrl } from './browserPolyfill';
+
+/*
+ * This just imports a blob of JS that New Relic give us so we can track usage/errors from real
+ * users. We should only need to upgrade it in the event New Relic makes a major change, which
+ * they'll let us know about.
+ */
+import { newrelicBrowserJS } from './newrelicBrowser';
+
+/**
+ * This component wraps all pages on the website, and through [Helmet](https://github.com/nfl/react-helmet/)
+ * sets up base CSS, favicons, javascript, and our New Relic RUM/error reporting (see `newrelicBrowserJS` above).
+ *
+ * @module PageWrap
+ */
+class PageWrap extends React.Component {
+	/**
+	 * This method ensures important app state props passed
+	 * from `AppContainer` are passed to children (feature containers)
+	 *
+	 * @method renderChildren
+	 * @returns {Array} Children with mapped props
+	 */
+	renderChildren() {
+		const { self, localeCode, location } = this.props;
+
+		return React.Children.map(this.props.children, (child, key) =>
+			React.cloneElement(child, { self, localeCode, location, key })
+		);
+	}
+
+	renderHeadLinks(assets) {
+		return assets.map((props, index) =>
+			<link key={`${props.rel}-${index}`} {...props} />
+		);
+	}
+
+	/**
+	 * @return {React.element} the page wrapping component
+	 */
+	render() {
+		const { localeCode, css, iconSprite, swarmLogos } = this.props;
+
+		// Parse localeCode for ISO 639-1 languages code.
+		// (ie. 'en', 'it', etc)
+		// @see https://github.com/meetup/swarm-sasstools/blob/master/scss/utils/helpers/_i18n.scss
+		const lang = localeCode.substring(0, 2);
+
+		return (
+			<div
+				id="root"
+				className={`column lang_${lang}`}
+				style={{ minHeight: '100vh' }}
+			>
+				<Helmet defaultTitle="Meetup" titleTemplate="%s - Meetup">
+					{css && this.renderHeadLinks(css)}
```

Comment:
I think we can make this component simpler and easier to work with by avoiding having separate `css`, and `swarmLogos` props - instead, what do you think about just accepting a single `<Helmet>` element as a prop, which can contain whatever _additional_ `<head>` content that an application needs? This `PageWrap` component would then render its 'standard'/base `<Helmet>` content, and then render the passed-in Helmet `prop` as a sibling - I believe that the contents of both elements will be merged correctly by Helmet.

The benefits of that approach are that `PageWrap` can be simpler because it doesn't have to parse two different props using `renderHeadLinks`, and the parent component can have as much custom control over `<head>` as it wants without ever having to modify the `<PageWrap>` interface to support more props.

```jsx
// parent component
const head = (
  <Helmet>
    <link ...swarm stuff.../>
    <link ...swarm stuff.../>
    <link ...css stuff.../>
  </Helmet>
);
return (
  <PageWrap head={head}>
    ...
  </PageWrap>
);
```